### PR TITLE
Improve the (already awesome) readme

### PR DESCRIPTION
**Pull request by klippx**
Hi!

Thanks for this wonderful gem which I finally had the pleasure to work with after hearing about it for 1.5 years (Rails conf San Diego 2014 baby!).

When I got started these are the things that (if added to the readme) would have helped me get up and running even quicker. Basically - remove unnecessary stuff, require the correct stuff, skip unnecessary inheritance, explain where magic variables are coming from, etc.

Hope you like it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -244,11 +249,22 @@ end
...
+  def initialize(name: name)
+    @name = name
+    @percent_enabled = 100
+  end
+
   def enabled?
     percent_enabled > 0 && rand(100) < percent_enabled
   end
+
+  # ...
```
Comment (bryongloden) [inline README.md]: Nice. How are you <3

File: README.md [hunk 2]
```diff
@@ -40,9 +40,9 @@ class MyWidget
   include Scientist
 
   def allows?(user)
-    science "widget-permissions" do |e|
-      e.use { model.check_user(user).valid? } # old way
-      e.try { user.can?(:read, model) } # new way
+    science "widget-permissions" do |experiment|
```
Comment (natesholland) [inline README.md]: 👍

File: README.md [hunk 3]
```diff
@@ -55,22 +55,25 @@ If you don't declare any `try` blocks, none of the Scientist machinery is invoke
...
   def enabled?
     # see "Ramping up experiments" below
-    super
+    true
   end
 
   def publish(result)
     # see "Publishing results" below
-    super
+    Rails.logger.info result
```
Comment (natesholland) [inline README.md]: I don't think it's safe to assume that Rails will be defined in the scope. Scientist could be run in the scope of Sinatra and rails is not listed as a gem dependency.

File: README.md [hunk 4]
```diff
@@ -191,9 +194,11 @@ end
 And this cleaned value is available in observations in the final published result:
 
 ```ruby
-class MyExperiment < ActiveRecord::Base
+class MyExperiment
   include Scientist::Experiment
 
+  # ...
```
Comment (natesholland) [inline README.md]: What is this comment for?

Comment (klippx) [inline README.md]: @natesholland it is the folded part of the experiment boiler plate (a representation thereof)

```
  attr_accessor :name

  def initialize(name: name)
    @name = name
  end
```

File: README.md (hunk 3 above)
Comment (klippx) [inline README.md]: @natesholland Yes I can agree and I was concerned with that myself. Any suggestion? Simply `@logger.log result` ?

**Comment by strand** [comment]
Is there anything blocking this PR?

File: README.md (hunk 3 above)
Comment (jbarnette) [inline README.md]: I think just like `p result` is fine for an example.

File: README.md [hunk 5]
```diff
@@ -244,11 +249,22 @@ end
 As a scientist, you know it's always important to be able to turn your experiment off, lest it run amok and result in villagers with pitchforks on your doorstep. In order to control whether or not an experiment is enabled, you must include the `enabled?` method in your `Scientist::Experiment` implementation.
 
 ```ruby
-class MyExperiment < ActiveRecord::Base
+class MyExperiment
```
Comment (jbarnette) [inline README.md]: I like how the old example shows that `percent_enabled` can be persisted in a DB, but this is cleaner.

**Comment by klippx** [comment]
*tumbleweed*

File: README.md (hunk 3 above)
Comment (jesseplusplus) [inline README.md]: @klippx if change this line to remove the reference to Rails, I'm happy to get this merged today 😃

**Comment by klippx** [comment]
Updated with requested changes!

**Comment by jesseplusplus** [comment]
🙇‍♀️ thanks @klippx! Merging now.

**Pull request merged**